README.md

```diff
@@ -1,30 +1,62 @@
 # Spin Cloud GPU
 
-A plugin that allows you to develop your AI enabled app locally while using GPUs on Fermyon Cloud.
+The `spin cloud-gpu` plugin allows you to use GPUs on Fermyon Cloud while running your Spin app locally with `spin up`. 
 
-## Usage
+Ever tested your AI-powered Spin app locally and spent quite a while waiting for your machine's compute to kick in...? Us too! That's why we built this plugin. Now with a few simple steps, you can use Fermyon Serverless AI GPUs to run inferencing and embedding requests for locally running Spin applications. Optionally, once local testing is complete and you're ready to run your application at scale, you can deploy to Fermyon Cloud with the standard `spin cloud deploy` command.  
+
+### How it works
+
+The `spin cloud-gpu` plugin, will be deploying a Spin application to Fermyon Cloud on your behalf that will serve as a proxy to access GPUs for your local Spin application. With the `spin cloud-gpu init` command, you'll implicitly deploy a Fermyon Cloud Spin application that will act as a proxy to access Fermyon Cloud GPUs from your local SPin application. You can always delete the proxy either via Fermyon Cloud UI or via `spin cloud-gpu destroy`. Read on to learn more about the prerequisites and the specific commands. 
+
+## Prerequisites 
+
+* You'll need a [Fermyon Cloud account](cloud.fermyon.com) if you don't have one already! Don't worry, this feature is available on our Developer Plan (free of charge). 
+* Because you're accessing Fermyon Cloud's Serverless AI, you'll need to be enrolled in private beta. You can request access to the private beta with this [form](https://fibsu0jcu2g.typeform.com/to/mNzgXRvB).
+* A local directory with a Spin application you'd like to connect to Fermyon Serverless AI. If you don't have one, you can create one via `spin new`
+
+> Note that use of the `cloud-gpu` feature will count against your Fermyon Cloud quotas, specifically your [Spin app count](https://developer.fermyon.com/cloud/faq#quota-limits) quota as well as your [inferencing request](https://developer.fermyon.com/cloud/serverless-ai#quotas-and-service-limitations-for-fermyon-serverless-ai) quota. 
+
+
+## Installation Steps
+
```

Comment:
Hey @skabber. Thanks for this contribution! The `create_plugin.sh` script is meant for local development. 

For users in general, they can install the plugin without having to build it locally by using the following command. 

```bash
spin plugins install -u https://github.com/fermyon/spin-cloud-gpu/releases/download/canary/cloud-gpu.json -y
```

The above command installs prebuilt binaries that are built in CI.